sentry-core/src/backtrace_support.rs

```diff
@@ -1,3 +1,7 @@
+//! Backtrace utilities for sentry
+//!
+//! Exposes functions to capture, process and convert/parse stacktraces
```

Comment:
_nit_: Would be dope to convert the example into a doc test, WDYT? That way, we have a more self-explaining docs.rs page and it's automatically tested to compile etc as well. For now, there don't have to be any assertions.

Reply:
I plan to move this out into its separate crate anyway, once all dependants are also moved out. I will keep this in mind ;-)